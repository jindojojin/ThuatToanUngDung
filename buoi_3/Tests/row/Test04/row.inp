50
6 8 5
26 28 4
25 22 2
10 16 4
19 29 3
26 13 4
1 18 5
17 12 3
5 1 5
11 18 2
8 28 1
25 26 1
9 7 5
6 7 4
10 14 2
16 4 1
27 29 4
22 22 3
20 14 5
4 23 3
13 9 5
13 12 4
16 5 1
25 24 2
13 27 1
16 2 4
1 7 1
26 7 3
27 8 3
15 10 1
19 23 4
24 1 5
29 3 1
27 4 2
12 16 1
5 1 1
7 12 4
3 27 5
12 6 3
7 14 3
8 20 1
11 20 3
24 6 5
9 25 5
10 18 5
18 22 4
12 2 2
24 22 1
9 9 1
6 25 4
100
9 47 2
26 37 1
7 45 2
6 48 3
28 36 1
19 50 3
13 47 2
13 27 3
13 45 2
19 22 5
23 37 4
17 44 3
20 48 1
0 3 2
0 8 1
18 41 5
29 48 5
31 42 1
23 26 4
16 23 2
7 28 2
6 50 4
7 15 3
17 28 2
13 34 4
23 47 1
0 32 1
12 28 3
35 49 1
41 46 5
5 34 1
0 21 4
22 38 5
5 49 5
5 14 1
34 50 5
25 39 4
10 50 1
6 31 3
1 32 5
18 24 2
44 45 1
24 47 3
30 45 2
8 27 1
3 39 3
24 27 1
8 23 5
2 29 2
12 37 2
6 26 2
22 34 3
29 30 2
13 17 5
32 45 5
6 29 3
8 10 3
0 19 5
3 10 1
23 26 4
7 29 4
7 40 3
14 32 5
23 38 5
14 16 4
31 32 2
10 30 3
17 32 1
18 34 2
6 49 2
38 41 3
12 31 5
35 48 3
6 30 3
7 13 5
19 23 4
2 40 1
33 35 2
10 32 3
12 39 2
19 21 5
32 44 2
6 31 4
12 40 4
12 15 2
5 27 3
10 22 2
2 47 1
40 48 3
7 28 2
3 42 3
22 33 1
12 30 5
0 12 3
10 34 2
12 49 1
32 39 3
22 37 3
0 33 1
1 25 1
