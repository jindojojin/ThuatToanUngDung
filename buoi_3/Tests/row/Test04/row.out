1070
441
1046
1568
434
880
872
576
848
117
553
1364
1560
550
0
502
654
444
32
0
938
2080
204
600
1450
952
1526
758
774
549
1613
2051
105
1137
874
630
553
2579
1178
507
600
0
788
108
1376
1801
358
465
1488
848
938
398
0
0
630
1178
0
414
0
32
1013
1348
421
105
0
0
974
588
708
1070
170
460
390
1178
68
156
1810
108
974
848
117
108
1450
1534
140
780
140
1970
220
938
2119
675
460
755
848
1705
72
398
1613
1369
